zgrab_schema.py

```diff
@@ -337,11 +337,16 @@
     "headers":zgrab_http_headers
 })
 
+zgrab_http_request_response = SubRecord({
+    "request":zgrab_http_request,
+    "response":zgrab_http_response
+})
+
 zgrab_http = Record({
     "data":SubRecord({
       "http":SubRecord({
-        "request":zgrab_http_request,
-        "response":zgrab_http_response
+        "response":zgrab_http_response,
+        "request_response_chain":ListOf(zgrab_http_request_response)
```

Comment:
That looks good. The schema file is a DSL called zschema. You can install it with `pip install zschema` and then have it validate that a JSON file matches a given schema. Ask @zakird or @amirian28 for more details about how to use it, it's quite possibly the least documented tool that we have.